src/components/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useEffect, useState } from 'react';
+import { getPeople } from '../../api';
+import { Person } from '../../types';
+import { PeopleTable } from '../PeopleTable/PeopleTable';
+
+export const PeoplePage: React.FC = () => {
+  const [people, setPeople] = useState<Person[] | null>(null);
```

Comment:
just use empty array as initial value
```suggestion
  const [people, setPeople] = useState<Person[]>([]);
```